### PR TITLE
Added stage to make blazegraph journal

### PR DESCRIPTION
**Pull request by justaddcoffee**

### CODE REVIEW

**Comment by justaddcoffee** [comment]
partially addresses #134

File: Jenkinsfile [hunk 1]
```diff
@@ -76,7 +76,22 @@ pipeline {
...
+            steps {
+                dir('./config/blazegraph') {
+                        git(
+                                url: 'https://github.com/balhoff/blazegraph-runner.git',
+                                branch: 'master'
+                        )
+                        sh 'sbt stage'
+                        sh 'cd ..'
+                        sh 'pigz -d ../merged-kg.nt.gz'
+                        sh './blazegraph-runner/target/universal/stage/bin/blazegraph-runner/blazegraph-runner load --informat=ntriples --journal=merged-kg.jnl --use-ontology-graph=true ./merged-kg.nt'
```
Comment (balhoff) [inline Jenkinsfile:90]: @justaddcoffee I may be missing something but I think this path should just be `./blazegraph-runner/target/universal/stage/bin/blazegraph-runner`.

Comment (justaddcoffee) [reply Jenkinsfile:90]: Thanks @balhoff ! You are right

**Comment by justaddcoffee** [comment]
@balhoff thanks for your help, this PR successfully creates a blazegraph journal. 

Might eventually break this into a separate (or optional) jenkins job b/c the runtime is a about a day now, but not a dealbreaker for me

**Pull request merged**